LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Computational Problem Solving & Software Development, DCU
```

Comment:
isn't it copyright to us ("the Rabble developers" or something) and not to DCU?

Reply:
That's tru, it was autofilled